jujugui/static/gui/src/app/models/models.js

```diff
@@ -2472,15 +2490,17 @@ YUI.add('juju-models', function(Y) {
         var charmOptions = charm.get('options');
         var serviceName = service.get('id');
 
-        // Exclude this service if it is a ghost or if it is named "juju-gui".
-        // This way we prevent the Juju GUI service to be exported when the
-        // bundle is created from a live environment. Note that this is a weak
-        // check: in theory, each deployed charm can be named "juju-gui", but
-        // we still assume this convention since there are no other (more
-        // solid) ways to exclude the Juju GUI service.
-        if (service.get('pending') === true ||
-            serviceName === JUJU_GUI_APPLICATION_NAME) {
-          return;
+        if (!includeUncommitted) {
+          // Exclude this service if it is a ghost or if it is named "juju-gui".
+          // This way we prevent the Juju GUI service to be exported when the
+          // bundle is created from a live environment. Note that this is a weak
+          // check: in theory, each deployed charm can be named "juju-gui", but
+          // we still assume this convention since there are no other (more
+          // solid) ways to exclude the Juju GUI service.
+          if (service.get('pending') === true ||
+          serviceName === JUJU_GUI_APPLICATION_NAME) {
```

Comment:
I was actually thinking we should remove this check. Now that the GUI is included by default in the model if the user has added the GUI it's an explicit part of their model.